README.md

```diff
@@ -161,7 +161,7 @@ You have three different options:
 1. Using `yarn environment`:
 
    There is a script to facilitate the generation of the environment variables inside of this repository.
-   This script is called `./scripts/genesisEnvironmentVariables.ts`.
+   This script is called `./launchScripts/genesisEnvironmentVariables.ts`.
```

Comment:
Why is this changing to launch scripts?

Reply:
Why isn't it called scripts?

Reply:
It is a remanence of last PR #82. I forgot to update the Reame there.

Reply:
it is just to make more clear that those scripts are just needed during the projects set-up.